pom.xml

```diff
@@ -3,8 +3,7 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
-    <packaging>war</packaging>
+    
     <groupId>jstat.gui</groupId>
     <artifactId>jstatgui</artifactId>
     <version>0.0.1</version>
```

Comment:
pls use the pom file we have in the master branch.

Reply:
yes I should have already done that